src/components/Login/Login.js

```diff
@@ -0,0 +1,124 @@
+import React from 'react';
+import './Login.scss';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faFacebook, faGoogle } from '@fortawesome/free-brands-svg-icons';
+import { useState } from 'react';
+
+const Login = ({ data, closeModal }) => {
+  const { title, subtitle, facebook, google, btn, text, footer } = data;
+  const [loginInfo, setLoginInfo] = useState({
+    email: '',
+    password: '',
+  });
+  const [idCheck, setIdCheck] = useState('');
+  const [pwCheck, setPwCheck] = useState('');
+
+  const saveLoginInfo = e => {
+    const { name, value } = e.target;
+    setLoginInfo({ ...loginInfo, [name]: value });
+  };
+
+  const idCondition =
+    loginInfo.email.includes('@') &&
+    loginInfo.email.length > 5 &&
+    loginInfo.email.includes('.');
+
+  const IdChecked = () => {
```

Comment:
```suggestion
  const idChecked = () => {
```
PascalCase는 컴포넌트 네이밍에 쓰입니다
함수는 camelCase로 작성 해 주세요~

Reply:
수정 완료입니다.